tests/integration/targets/meraki_mr_radio/tasks/main.yml

```diff
@@ -0,0 +1,146 @@
+# Test code for the Meraki modules
+# Copyright: (c) 2021, Tyler Christiansen (@supertylerc)
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- block:
+  - name: Create network with type wireless
+    meraki_network:
+      auth_key: "{{ auth_key }}"
+      state: present
+      org_name: "{{ test_org_name }}"
+      net_name: IntTestNetworkWireless
+      type: wireless
+      timezone: America/Chicago
+    delegate_to: localhost
+
+  - name: Create RF profile
```

Comment:
Also, please add a task which adds the access point to the test network.

Reply:
Resolved in d89deab.